TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ReservationDao.java

```diff
@@ -1,31 +1,65 @@
 package cz.muni.fi.travelAgency.dao;
+
 import cz.muni.fi.travelAgency.entities.Customer;
-import cz.muni.fi.travelAgency.entities.Excursion;
 import cz.muni.fi.travelAgency.entities.Reservation;
 import cz.muni.fi.travelAgency.entities.Trip;
-
```

Comment:
Here the empty lines usually have some meaning. Like they divide imports from inside the project and for example java utils. No need to add them back, just FYI.